src/videojs-contrib-hls.js

```diff
@@ -547,7 +547,7 @@ class HlsHandler extends Component {
       this.ignoreNextSeekingEvent_ = true;
     });
 
-    this.setupQualityLevels_();
+    this.tech_.ready(this.setupQualityLevels_.bind(this));
```

Comment:
Ha, I used an arrow function then changed it to `bind` because I saw it used elsewhere.